activerecord/test/cases/create_new_entry.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
```

Comment:
Actually due to its name I don't think CI executes this file as part of the tests run. I believe the suite only looks for `*_test.rb` files

Reply:
Great catch! I moved my test into persistence_test.rb